docs/3. Product Features/06. Templates.md

```
@@ -127,25 +127,45 @@ Go to `Chat` window and to the particular contact to whom you want to send messa
-
-1. Create a template in English and get it approved.
-1. Create a template in another language like - Hindi. Use the same title.
-1. Now when adding the template in flow, you can add English template in default language flow. If contact selects a different language (Hindi), then the Hindi version of HSM will be send to the contact. It chooses the language specific HSM automatically based on the selection of language by the end-user.
+Steps 1: Create and Approve English Template
+   1. Click “Create Template.”
+   2. Fill in the template in English, with a unique element name. 
+   3. Choose category (e.g., Utility, Marketing).
+   4. Submit the template for approval.
+   5. Wait until it is approved by WhatsApp.
 
 **English**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)